test/test_helper.rb

```diff
@@ -84,7 +84,7 @@ def self.get_file(url)
     if not @cache.keys.include? url
       @cache[url] = Tempfile.new('APIMatic')
       @cache[url].binmode
-      @cache[url].write(open(url, {ssl_verify_mode: OpenSSL::SSL::VERIFY_NONE}).read)
+      @cache[url].write(File.open(url, 'rb').read)
```

Comment:
since it's test - bumping up the [version](https://github.com/Moesif/moesifapi-ruby/blob/master/moesif_api.gemspec#L3) is not required.